coordinator/ts/subgraph/types.ts

```diff
@@ -0,0 +1,41 @@
+import type { ESupportedNetworks } from "../common";
```

Comment:
should the enum be imported as value vs as a type?

Reply:
Here no, because I use it as type. But if I need to access values (ex: ESupportedNetworks.SEPOLIA), in this case type import won't work.

Reply:
ohhhh interesting, thanks for explaining :)